### PR TITLE
[cryptolib] Add a top-level `otcrypto.h` header

### PR DESCRIPTION
**Pull request by cfrantz**
Add a top-level header file that includes all other cryptolib headers.

### CODE REVIEW

**Comment by cfrantz** [comment]
@vsukhoml Does this header work for you?

**Review by jadephilipoom — APPROVED** [review]

**Comment by vsukhoml** [comment]
yes, looks good!

File: sw/device/lib/crypto/BUILD [hunk 1]
```diff
@@ -34,6 +34,15 @@ cc_import(
     ],
 )
 
+cc_import(
```
Comment (jadephilipoom) [inline sw/device/lib/crypto/BUILD:37]: nit: I think a comment here might be helpful to clarify the contexts in which this target should (or should not) be used. Is it only for tests within the OT repo of the freestanding/exported lib?

**Review by jadephilipoom — APPROVED** [review]

File: sw/device/lib/crypto/BUILD (hunk 1 above)
Comment (cfrantz) [reply sw/device/lib/crypto/BUILD:50]: Fixed: added a comment.

This target is used to test that the freestanding version of the library & headers work correctly on their own, whereas the `:crypto` target can be used to test cryptolib within the codebase.

**Pull request merged**